.pre-commit-config.yaml

```diff
@@ -3,6 +3,13 @@ default_install_hook_types:
   - commit-msg
 
 repos:
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.5.6
```

Comment:
We should use the latest from main (free-floating), to be inline with the `requirements.txt`. This is to avoid discrepancies between local and CI ruff checks